### DISCUSSION TITLE
Can we bin the UMB_MCULTURE cookie?

### DISCUSSION THREAD

**Discussion by drpeck** [umbraco/Umbraco-CMS#11117] [Features and ideas]
The UMB_MCULTURE cookie is being flagged in some security scans as being insecure (not HttpOnly or SameSite="Strict/Lax"), when in fact it is only ever set in JavaScript. Would local storage not be a better store than a cookie? That would reduce the request size and avoid confused InfoSec.

https://github.com/umbraco/Umbraco-CMS/blob/dev/v9/src/Umbraco.Web.UI.Client/src/navigation.controller.js

**Comment by markadrake** [comment]
`UMB_MCULTURE` is a legacy/under-documented cookie. I was looking at it today, wondering what it's used to track.
